### ISSUE TITLE
It says invalid string position

### ISSUE THREAD

**Issue by muneymercer** [laurence-myers/flspo#2]
This are my inputs:

```
C:\Users\(myuser)>cd C:\flspo-v1.0.0

C:\flspo-v1.0.0>flspo.exe
invalid string position

```

**Comment by laurence-myers** [comment]
D'oh! Sorry, this is one of my first attempts at writing an app in C++, so I'm sure there'sa few bugs. I'll look into it.

Looks like it's failed reading the .nfo files. At this point I'm not sure if it's a problem when reading a particular file, or just walking through the directories. I'll get you a new version of the .exe with verbose output, that should at least narrow down where the problem lies.

**Comment by laurence-myers** [comment]
@muneymercer, Please try running this version: 
[flspo-v1.0.0-verbose-debug.zip](https://github.com/laurence-myers/flspo/files/8348318/flspo-v1.0.0-verbose-debug.zip)

Here's the changes I made: https://github.com/laurence-myers/flspo/commit/50c3db1c5f015bdf9a0aeeb2eedafd13f73e0aba

It will print out more info. This should help pinpoint the source of the error.

Example output looks like below:

```
"C:\\Users\\...\\Documents\\Image-Line\\FL Studio\\Presets\\Plugin database\\Installed\\Generators\\VST3\\Wurli V2.nf
o"
Key: fileversion
Value: 2
Key: iconindex
Value: 48
Key: color
Value: $40B6F6
Key: tip
Value: VST+VST3 / 64 bit
Key: ps_name
Value: Wurli V2
(... etc ...)
Key: ps_file_vendorname_1
Value: Arturia
Key: ps_file_category_1
Value: Instrument|Synth
name: Wurli V2 path: "C:\\Users\\...\\Documents\\Image-Line\\FL Studio\\Presets\\Plugin database\\Installed\\Generato
rs\\VST3\\Wurli V2.nfo" pluginType: 2 summary: VST+VST3 - 64 bit
```

There could be a lots of info. I'm only interested in the output for the last file, which should be the cause of the problem. Please provide the program's output, then I'll investigate further.

(In testing, I discovered that a directory name containing emoji will not work. This indicates that Unicode isn't properly supported. I've raised this as issue #3.)

**Comment by laurence-myers** [comment]
I think I've found the issue. Could you please try this version, and let me know if it works?

[flspo-issue-2-fix.zip](https://github.com/laurence-myers/flspo/files/8348461/flspo-issue-2-fix.zip)

flspo was looking for an entry in the .nfo file like "tip=VST / 32+64 bit", and would try to replace the "/" with a "-". If "/" was missing it would error. I've removed the logic to read that entry, since it's not actually used.

_(This branch has the code changes: https://github.com/laurence-myers/flspo/commits/issue-2-invalid-string-position)_

**Comment by muneymercer** [comment]
Hey, this is what I got this time. 
![flspo error](https://user-images.githubusercontent.com/70086518/160247684-ae558d57-ad61-4bf9-a397-9bf8cabff840.JPG)

**Comment by laurence-myers** [comment]
D'oh! My bad. I uploaded a "debug" version, which needs the Windows SDK or Visual Studio installed, which wasn't my intention. Please try this "release" version.

[flspo-issue-2-fix-2.zip](https://github.com/laurence-myers/flspo/files/8356625/flspo-issue-2-fix-2.zip)

**Comment by laurence-myers** [comment]
Fixed in [v1.1.0](https://github.com/laurence-myers/flspo/releases/tag/v1.1.0).
